vectors/cryptography_vectors/asymmetric/DH/bad_exchange.txt

```diff
@@ -0,0 +1,19 @@
+# This are pairs of DH vectors that an exchange between them will result in an error.
```

Comment:
s/this/these

Please add a comment about what type of failure each one causes and that these were custom generated.

This vector file also needs an entry in `test-vectors.rst`.